.github/workflows/reusable.rebuild.yml

```diff
@@ -252,14 +256,14 @@ jobs:
     - name: Build LLVM
       shell: cmd
       run: |
-        python ./alloy.py -b --version=${{ inputs.version }} --verbose --generator="${{ inputs.vs_generator }}"
-        cd alloy_results* && type alloy_build.log
+        cmake %ISPC_HOME%\superbuild -B build-${{ inputs.version }} --preset os -DLLVM_VERSION=${{ inputs.version }} -DCMAKE_INSTALL_PREFIX=%LLVM_HOME%\bin-${{ inputs.version }} -DBUILD_STAGE2_TOOLCHAIN_ONLY=ON -DXE_DEPS=OFF -DLLVM_DISABLE_ASSERTIONS=OFF
```

Comment:
Wrong slash is used:
- `%ISPC_HOME%\superbuild`
- `%LLVM_HOME%\bin-${{ inputs.version }}`